src/Wallabag/CoreBundle/Twig/WallabagExtension.php

```diff
@@ -16,6 +28,27 @@ public function removeWww($url)
         return preg_replace('/^www\./i', '', $url);
     }
 
+    public function getGlobals()
```

Comment:
I want to have @j0k3r opinion about this new method. 
Do we need test for it? Because I only use `getBuilderFor*ByUser`, already tested.